api/views.py

```diff
@@ -69,8 +69,12 @@ def post_message():
     return_schema = PostedMessageSchema()
 
     notifications_repo = NotificationsRepo(current_app.config['NOTIFICATIONS_REPO_CONF'])
-    use_case = use_cases.PublishNewMessageUseCase(notifications_repo)
-    use_case.publish(message)
+    use_cases_list = [
+        use_cases.PublishNewMessageUseCase(notifications_repo),
```

Comment:
Just the status change use case - the publish new message use case will be triggered by the observer seeing a new message in the shared db from a different channel endpoint

Reply:
right